MonoGame.Framework.Content.Pipeline/H264Importer.cs

```diff
@@ -16,5 +19,10 @@ public override VideoContent Import(string filename, ContentImporterContext cont
             var content = new VideoContent(filename);
             return content;
         }
+
+        public override VideoContent Import(Stream input, string virtualFilename, ContentImporterContext context)
+        {
+            throw new NotSupportedException("Video content must be stored on disk, not imported from a stream.");
```

Comment:
Incorrect assumption for AudioContent.  Audio content, including compressed, is loaded into memory before exporting in the target format.